downloader/src/main/kotlin/vcs/GitRepo.kt

```diff
@@ -194,7 +194,7 @@ class GitRepo : VersionControlSystem(), CommandLineTool {
             // Switching manifest branches / revisions requires running "init" again.
             runRepo(workingTree.workingDir, "init", *manifestOptions.toTypedArray())
 
-            // Repo allows to checkout Git repositories to nested directories. If a manifest is badly configured, a
+            // Repo allows to check out Git repositories to nested directories. If a manifest is badly configured, a
```

Comment:
Nit (sorry for not mentioning this earlier): Similar to the other commits that fix the same type of inspection hint, this could also be squashed to the previous commit as a general "typo fixes in docs" commit.

Reply:
Would you agree that this nit is actually quite a bit subjective? E.g. it could be squashed, but it also could remain as is? So, is your comment optional or a must fix?

Reply:
I agree it's subjective, but it would follow the idea to fix the same kind of inspection hint in one commit throughout the code base. As you followed that rationale earlier, I thought you're fine with doing it here, too. But it's up to you.

Reply:
I feared if I follow it again I may end-up with some unspoken agreement to always do this in the future, e.g. go through my commit stash and see what typo fixes can be squashed. So, I feared that this might introduce additional effort where I don't see always a value in it.